network-api/networkapi/wagtailpages/tests/research_hub/test_detail_page.py

```diff
@@ -33,6 +36,60 @@ def test_research_detail_page_breadcrumbs(self) -> None:
         self.assertEqual(len(breadcrumbs), 2)
         self.assertEqual(breadcrumbs, expected_breadcrumbs)
 
+    def test_get_research_authors(self) -> None:
```

Comment:
Here, it would be interesting to add a `Profile` linking to another page, and check that it doesn't show up in our `research_authors`.

Reply:
Updated! See comment below